src/backend/utils/adt/json.c

```diff
@@ -338,12 +354,67 @@ datum_to_json(Datum val, bool is_null, StringInfo result,
 			break;
 		default:
 			outputstr = OidOutputFunctionCall(outfuncoid, val);
+			// The outfuncoid are specific to babelfish and therefore no macro is defined.
```

Comment:
please state which data types these are referring to - sys.DATETIME, etc

Reply:
Resolving all the comments as just moving declaration from parser.h to gram-tsql-prologue.y.h